Packs/Jizo_M/Integrations/JizoM/JizoM.yml

```diff
@@ -0,0 +1,282 @@
+category: Network Security
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: JizoM
+  version: -1
+name: JizoM
+display: JizoM
+description: This integration ensures interaction with the JizoM API.
+configuration:
+- section: Collect
+  display: Username
+  displaypassword: Password
+  name: credentials
+  type: 9
+  required: true
+  additionalinfo: Reliability of the source providing the intelligence data.
```

Comment:
What is the additionalinfo here? This information seems unrelated to the username & password parameter.

Reply:
you are right. I removed it.